project_one.html

```diff
@@ -168,29 +252,31 @@ <h2 class="about_h">Myself</h2>
           </ul>
         </div>
         <a href="#" class="pro"> Get my resume </a>
-        <div class="">
+      </div>
+        <div class="about_right">
           <ul>
             <li>
               Langguages <img class="iconimg" src="./icons/Union.png" alt="" />
 
               <ul class="lang">
                 <li class="ski one"><span class="sk"> JavaScript</span></li>
-                <li class="ski two"><span class="sk"> HTML </span></li>
-                <li class="ski three"><span class="sk"> CSS </span></li>
+                <li class="ski two"><span class="sk sk1"> HTML </span></li>
+                <li class="ski three"><span class="sk sk1"> CSS </span></li>
               </ul>
             </li>
           </ul>
           <ul>
             <li>Frameworks</li>
           </ul>
-          <hr />
+          <hr/>
           <ul>
             <li>Skills</li>
           </ul>
-          <hr />
+          <hr/>
```

Comment:
- [ ] I see there is an empty space to the right of the page which is causing a vertical scroll. Kindly remove the empty space and remove the vertical scroll as this does not create a good user experience.

![image](https://user-images.githubusercontent.com/43995830/160105663-1bf4bbcc-b4d5-4797-8d0f-3f3521b42e67.png)